server/nodejs/controllers/OffersService.js

```diff
@@ -320,17 +55,18 @@ exports.offernew = function(args, res, next) {
     { prop: "price" },
     { prop: "description" },
     { prop: "currency" },
-    { prop: "certguid", defaultValue: "" },
-    { prop: "paymentoptions", defaultValue: "SYS" },
-    { prop: "geolocation", defaultValue: "" },
-    { prop: "safesearch", defaultValue: "Yes" },
-    { prop: "private", defaultValue: "0" }
+    { prop: "cert_guid"},
```

Comment:
I noticed all of the original default values are gone.  Is this intentional?

Reply:
yes because it doesn't seem to do anything when you add them.  the default values in swagger UI stay the same (not so helpful).  so I figured to remove it to avoid confusion.  I may be missing something obvious though.  lmk if I need to update it

Reply:
Hot, all good.